### PR TITLE
Add accessibility page

### LINKED ISSUE #33: Update website #2 : update about page + accessiblity statement
PR what about the "about section":
- [x] update land ack.
- [ ] accessibility statement
    - (in past we haven't received many requests, which ones can we accom and how much notice do we need)

### PR DESCRIPTION
**Pull request by dcwalk**
First version based on our discussion, I reviewed our 2019 materials. We can add to this.

Part of https://github.com/ournetworks/2024/issues/33 (actually closes https://github.com/ournetworks/2024/issues/33 as I split off some items to a future issue to keep our website updates focused)

### CODE REVIEW

File: accessibility.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+order: 3
+title: "Accessibility"
+parent: "About the event"
+---
+
+We strive to reduce barriers that would prevent people from participating. If you have any access needs or questions, please contact [{{ site.email }}](mailto:{{ site.email }}) at least two weeks before the event.
+
+The details of this page are current as of {{ page.date }}.
```
Comment (garrying) [inline accessibility.md]: Is page.date showing up for you? It's blank locally. Maybe we need to build to see this?

File: accessibility.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Venue
```
Comment (garrying) [inline accessibility.md:13]:
```suggestion
## Venue
{:.font-display}
```

File: accessibility.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+312 Main Street has an accessible entrance, all gender accesible washrooms, and bike parking located inside the venue. We will be sharing more details here closer to the event.
+
+
+## Presentations
```
Comment (garrying) [inline accessibility.md:19]:
```suggestion
## Presentations
{:.font-display}
```

File: accessibility.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+For presentations we ask speaktes to consider the following suggestions to enhance the accessibility of any slides: 
+- use high-contrast slides with large fonts 
+- use short-form links for any URLS
+- provide a digital copy for people to follow along 
+
+If you would like to provide hard copies of materials the organizers can assist with printing, just let us know in advance of your talk. 
+
+Some guides with more information on preparing accessible presentations: 
+- https://www.w3.org/WAI/teach-advocate/accessible-presentations/
```
Comment (garrying) [inline accessibility.md]: Linkify these links?

**Comment by dcwalk** [comment]
Thanks @garrying I made the changes you suggested! Wanna take one more look?

**Comment by garrying** [comment]
Fixed a few things in [2a1c163](https://github.com/ournetworks/ournetworks.ca/pull/288/commits/2a1c1630832340da437680750989852f88e1ec19). Gonna merge

**Pull request merged**